### ISSUE TITLE
NSGMiner freezing

### ISSUE THREAD

**Issue by korishan** [MultiPoolMiner/MultiPoolMiner#1132]
Ok, so I thought it was the XMRig miner that was freezing, and under advice, I put
>setx XMRSTAK_NOWAIT=1
in the .bat file.

Well, turns out it's the NSGminer that's freezing. I get no text in the miner. I even tried to use the mouse to do a "quick edit" mode and select the screen to see if anything was there. Nope, no text at all.
Log file shows:

>2018-01-28 22:42:45 INFO: Starting AMD miner NsgminerAmd: 'C:\Mining\MultiPoolMiner\Bin\Nsgminer\nsgminer.exe --api-port 14028 --api-listen --neoscrypt -o stratum+tcp://neoscrypt.usa.nicehash.com:3341 -u 3JQbSU8asbq4gwhgQAbW7yrEivvxoFY44m.multipoolminer2 -p x --gpu-threads 1 --worksize 64 --intensity 15 --gpu-platform 0'
>....
>2018-01-28 22:43:20 ERROR: Failed to connect to miner (NsgminerAmd). 
>2018-01-28 22:43:21 INFO: Starting next run...
>...
>2018-01-28 22:44:25 ERROR: Failed to connect to miner (NsgminerAmd). 
>2018-01-28 22:44:25 INFO: Starting next run...
>...
> 2018-01-28 22:45:31 ERROR: Failed to connect to miner (NsgminerAmd). 
> 2018-01-28 22:45:32 INFO: Starting next run...

When it freezes like that, I can select the window and press "any key" and it'll close. However, when it's frozen, MPM won't close it and start another miner.

Thoughts, Suggestions, Ideas?????

Kori

**Comment by UselessGuru** [comment]
> However, when it's frozen, MPM won't close it and start another miner.

In this case remains your computer still responding?
Is MPM still looping?

Frozen miners are often a case of too much overclock. 
If you are having a problem with a miner then please report your issue on the developer page of the miner. MPM ist just starting and stopping miners.

**Comment by korishan** [comment]
Yeah, the computer still responds just fine. MPM seems to be still working just fine with the CPU miner.

I've only pushed the memory clocks slightly. They can't go very far as it is as I'm using older hardware. 

I have opened an issue with the miner author. I was just posting here to see if there might be a fix here where MPM would recognize the freeze and for it closed and after so many failed attempts, not try to use that one for awhile.
